.pre-commit-config.yaml

```diff
@@ -0,0 +1,48 @@
+# Install pre-commit hooks via
+# pre-commit install
+
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.3.0
```

Comment:
I think when using hooks from third parties, the same logic as for the GitHub actions applies and pinning to a specific commit is good practice. If you agree I'll do the work and suggest these changes. ;)